src/i_3dsound.c

```diff
@@ -315,13 +293,17 @@ static void I_3D_UpdateListenerParams(const mobj_t *listener)
         return;
     }
 
-    CalcListenerParams(listener, &lis);
-    I_OAL_UpdateListenerParams(lis.position, lis.velocity, lis.orientation);
+    if (!(menuactive && !netgame && !demoplayback) && !paused
```

Comment:
Could you add a comment here, like "only when listener is moving".

Reply:
Added here: https://github.com/fabiangreffrath/woof/pull/2215/commits/cdcb92519a5c7e4b00f202d13fd00866814a940b